### ISSUE TITLE
[doc issue] Typescript walkthrough missing dependency @ui5/builder

### ISSUE THREAD

**Issue by crater2150** [SAP-docs/sapui5#157]
### Issue description

When following the Typescript walkthrough, after "Step 2" it is no longer possible to run `npm start`, as it fails with the error
```
Failed to load task repository. Missing dependency to '@ui5/builder'? Error: Cannot find package '@ui5/builder' imported from <app root dir>/node_modules/@ui5/project/lib/graph/ProjectGraph.js
```

The problem seems to be fixable with `npm install @ui5/builder --save-dev`, which is not mentioned in the documentation. Should this work without that step?

### Feedback Type (Optional)

content gaps

### Page Title on SAP Help Portal (prefilled)

Step 2: Bootstrap (TypeScript)

### Page URL on SAP Help Portal (prefilled)

https://ui5.sap.com/#/topic/32b14d88bd484cd7b941aae37180f732

**Comment by KvM2** [comment]
Hi @crater2150 , thanks for your contribution! We'll look into it and get back to you.

**Comment by KlattG** [comment]
Hi @crater2150 ,  have you followed all the tutorial steps in the correct order? In Step 1 you are required to run
`npm install --save-dev @ui5/cli `.
This should have prevented the problem. Does the problem get fixed with this input?

**Comment by crater2150** [comment]
I just reran all commands from the tutorial on another machine, the error still occurs. I also re-ran `npm install --save-dev @ui5/cli`. Curiously, according to `package-lock.json`, the `@ui5/builder` package is installed, but in `node_modules/@ui5/cli/node_modules/@ui5/builder`, while installing it manually will put it in `node_modules/@ui5/builder`, thus fixing the problem. So this is probably a packaging issue, not a documentation issue.

I've attached `package-lock.pre.json` (state before running `npm install -D '@ui5/builder'`) and `package-lock.post.json` (state afterwards).
[package-lock.pre.json](https://github.com/user-attachments/files/16996794/package-lock.pre.json)
[package-lock.post.json](https://github.com/user-attachments/files/16996793/package-lock.post.json)

**Comment by crater2150** [comment]
Hi @flovogt,
As I said, I reran all commands from the tutorial on a different PC, so without a preexisting `package-lock.json`. I just noticed I mixed up the pre- and post manual install lockfiles, I'm sorry if that caused confusion.

But I have good news, when I tried to reproduce the issue just now, everything worked after step 2. I'm not sure what changed, the lock file only differs in some version numbers and I started from an empty directory like last time :shrug:
